cdc/processor/sinkmanager/manager.go

```diff
@@ -931,9 +975,24 @@ func (m *SinkManager) GetTableStats(span tablepb.Span) TableStats {
 	}
 	tableSink := value.(*tableSinkWrapper)
 
-	checkpointTs := tableSink.getCheckpointTs()
+	checkpointTs, version, advanced := tableSink.getCheckpointTs()
 	m.sinkMemQuota.Release(span, checkpointTs)
 	m.redoMemQuota.Release(span, checkpointTs)
+
+	stuckCheck := time.Duration(*m.changefeedInfo.Config.Sink.AdvanceTimeout) * time.Second
+	if advanced.After(time.Unix(0, 0)) && time.Since(advanced) > stuckCheck &&
```

Comment:
It looks `advanced.After(time.Unix(0, 0))` is always false ?

Reply:
`advance` can be equal to `time.Unix(0, 0)`, which means the table doesn't have a valid table sink at the time.